plugins/services/src/js/service-configuration/PodContainerArtifactsConfigSection.js

```diff
@@ -53,12 +65,15 @@ class PodContainerArtifactsConfigSection extends React.Component {
 
     return (
       <div>
-        <Heading level={3}>Container Artifacts</Heading>
+        <ConfigurationMapHeading level={3}>
+          Container Artifacts
+        </ConfigurationMapHeading>
         <ConfigurationMapTable
-          className="table table-simple table-break-word flush-bottom"
+          className="table table-simple table-align-top table-break-word flush-bottom"
           columnDefaults={{hideIfEmpty: true}}
           columns={this.getColumns()}
-          data={artifacts} />
+          data={artifacts}
+          editLink={this.getEditLink()} />
```

Comment:
I think this is implemented on an upcoming commit right? (https://github.com/dcos/dcos-ui/pull/1599/commits/caf5cb0d3117c45e1c0f491108da01dce64ca8e6#diff-5f5bab2424006fa57d80b4959f2a95adR142) I don't think this exists on the current branch yet, but being a property I don't think it hurts to have it around. 👍